resources/home/dnanexus/utils/io.py

```
@@ -0,0 +1,366 @@
+        output_name,
+        folder=f"{job_output}/igv_sessions",
+        tags=[expiry_date],
+        wait_on_close=True,
+    )
+
+    session_file_id = session_file.get_id()
+
+    return session_file_id
+
+
+def read_excluded_regions_to_df(file_id, project):
```

Ruff (0.8.0)

Missing return type annotation for public function `read_excluded_regions_to_df`

(ANN201)